alpha/nodejs-chart/values.yaml

```diff
@@ -0,0 +1,136 @@
+image:
+  ## Name of the image you want to build/deploy
+  ## Defaults to the release name
+  name: test-app
```

Comment:
I'm not sure if this makes sense, but the deploy passed if I used:

```shell
helm install test-app nodejs-chart
```

but failed when the name I gave on the install line did not match what was in the values.yaml file.  I would not have expected for them to have to match.

```
helm install test-helm nodejs-chart
```

Reply:
If I remove this line in the values.yaml it seems to be ok, so it's probably best not to include in the default file, and separately we probably need to figure out if we can make it work as an override.